solc/CommandLineInterface.cpp

```diff
@@ -790,7 +799,7 @@ void CommandLineInterface::handleCombinedJSON()
 	{
 		Json::Value contractData(Json::objectValue);
 		if (requests.count(g_strAbi))
-			contractData[g_strAbi] = dev::jsonCompactPrint(m_compiler->contractABI(contractName));
```

Comment:
Currently, the "compact json" has three members (abi, natspecdev, natspecuser) which are already serialised to strings, even before the "compact json" is.

I think we can safely introduce the option to pretty printing to the outside JSON, but shouldn't change those three fields as tools could expect them to be strings.